### PR TITLE
 Don't use default in typestr if we have cdb2cfg override

### PR DESCRIPTION
**Pull request by mohitkhullar**
Don't use default in typestr if we have cdb2cfg override

### CODE REVIEW

File: cdb2api/cdb2api.c [hunk 1]
```diff
@@ -927,8 +930,10 @@ void cdb2_set_comdb2db_config(const char *cfg_file)
                 cfg_file);
     memset(CDB2DBCONFIG_NOBBENV, 0, sizeof(CDB2DBCONFIG_NOBBENV) /* 512 */);
     if (cfg_file != NULL) {
+        cdb2cfg_override = 1;
         strncpy(CDB2DBCONFIG_NOBBENV, cfg_file, 511);
     } else {
+        cdb2cfg_override = CDB2CFG_OVERRIDE_DEFAULT;
```
Comment (mistachkin) [inline cdb2api/cdb2api.c]: It seems like this line should be moved into the reset_the_configuration() function?

File: cdb2api/cdb2api.c [hunk 2]
```diff
@@ -947,10 +952,12 @@ void cdb2_set_comdb2db_info(const char *cfg_info)
         CDB2DBCONFIG_BUF = NULL;
     }
     if (cfg_info == NULL) {
+        cdb2cfg_override = CDB2CFG_OVERRIDE_DEFAULT;
```
Comment (mistachkin) [inline cdb2api/cdb2api.c]: Same as above, line should probably move into the reset_the_configuration() function.

**Comment by mohitkhullar** [comment]
Thanks Joe

**Comment by riverszhang89** [comment]
Discussed with Mohit. No need to port it to cdb2jdbc.

**Pull request merged**